syncer.go

```diff
@@ -144,7 +143,7 @@ func NewSyncerFromFile(config *Config, clientConfig ClientConfig, bundle string,
 }
 
 func (s *Syncer) updateSuccessTimestamp() {
-	atomic.StoreUint64(&s.lastSuccessMonotonic, monotime.Now())
+	atomic.StoreUint64(&s.lastSuccess, uint64(time.Now().UnixNano()))
```

Comment:
I think it should be safe to update a time.Time here non-atomically if we're operating under the sync.Mutex, but we would have to make sure it's held in all update paths.

Reply:
Do we actually care about this being atomic, since we can just do `time.Since(s.lastSuccess).Unix()`? The only problem I see is that for the first sync we'll have time since the epoch 🤔

Reply:
👍